scrapers/Yanks.yml

```diff
@@ -0,0 +1,22 @@
+name: Yanks
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.yanks.com/trailers/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class='blocktitle clear']/h2/text()
+      Performers:
+        Name:
+          selector: //div[@class='blocktitle clear']/p/a/text()
+      Date:
+        selector: //div[@class='blog']/h4/text()
```

Comment:
I couldn't actually find a scene page where this selector worked in my tests though:

```
     Date:
        selector: //div[@class='blocktitle clear']/p/text()
        postProcess:
          - replace:
            - regex: .+(\d{4}-\d{2}-\d{2}).+
              with: $1
          - parseDate: 2006-01-02
```